2018-10-22 19:59:47,856 ERROR: Exception on /db [GET] [in /Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-10-22 19:59:52,192 ERROR: Exception on /db [GET] [in /Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-10-22 20:09:51,702 ERROR: Exception on /db [GET] [in /Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-10-22 20:09:57,418 ERROR: Exception on /db [GET] [in /Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-10-22 20:10:00,024 ERROR: Exception on /db [GET] [in /Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/Users/wmitchell/proj/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-10-22 20:10:18,780 INFO: Microblog startup [in /Users/wmitchell/proj/microblog/app/__init__.py:74]
